gateway/api/management/commands/update_jobs_statuses.py

```diff
@@ -15,7 +17,7 @@ class Command(BaseCommand):
 
     help = "Update running job statuses and logs."
 
-    def handle(self, *args, **options):
+    def handle(self, *args, **options):  #  pylint: disable=too-many-branches
```

Comment:
Due to this I would say that this is a good time to create an issue to try to refactor this method.

Reply:
New issue for it.  https://github.com/Qiskit-Extensions/quantum-serverless/issues/907